packages/build/tests/helpers/normalize.js

```diff
@@ -20,6 +20,7 @@ const NORMALIZE_REGEXPS = [
   [/\\/gu, '/'],
   [/[A-Z]:\//g, '/'],
   [/Program Files/gu, 'ProgramFiles'],
+  [/Users\/([^/])\//gu, 'Users/User'],
```

Comment:
```suggestion
  [/Users\/([^/]+)\//gu, 'Users/User/'],
```

This pattern works correctly,

Reply:
Nice catch!